### DISCUSSION TITLE
ISSUE: Cards Moving in List View

### DISCUSSION THREAD

**Discussion by ksanty** [community/community#8806] [Other Feature Feedback, Questions, & Ideas]
When I am adding labels in the list view, the status column moves the issue to another column. I checked workflows, but didn't see the issue coming from there.

**Comment by brianlovin** [comment]
Hey @ksanty I think this might be a bug report for the Issues feedback category here: https://github.com/github/feedback/discussions/categories/issues-feedback

If you cross-post there, the right people should see this and can help 😊
